tools/azsdk-cli/docs/specs/0-scenario-1.spec.md

```diff
@@ -379,41 +378,43 @@ Updating package metadata for Health Deidentification SDK...
 Package metadata updated for 5/5 languages
 ```
 
-### 4. Run PR Checks
+### 4. Run Validation Checks
 
-> Sample: will be changed when the spec for this tool is added
 
 **Command:**
 ```bash
-azsdk run-pr-checks --service healthdataaiservices --languages .NET,Java,JavaScript,Python,Go
+# Run all checks (default)
+azsdk package run-checks --package-path <path_to_sdk_package>/
```

Comment:
Just to make sure I'm understanding each call to the tool does checks for one language, right?